person.rb

```diff
@@ -17,4 +19,10 @@ def can_use_services?
   def _age?
     @age >= 18
   end
+
+  public
+
+  def correct_name
+    @name
+  end
```

Comment:
- [ ] [OPTIONAL] Great job placing the methods `private` and `public` visibility for some of the methods, however,  they are not necessary in this case. Methods are public by default unless specified otherwise.

- [ ] [OPTIONAL] Also, it will be nice to move the `attr_reader` and `attr_accessor` declarations to the top of the class for better visibility.